README.md

```diff
@@ -24,11 +24,11 @@
 
 <h3 align="center">
    &emsp;
-   <a href="https://discourse.floxdev.com"><b>Discourse</b></a>
+   <a href="https://discourse.flox.dev"><b>Discourse</b></a>
```

Comment:
Let's double-check that discourse will move.

Reply:
Didn't we say we are migrating to `flox.dev` domain? That means everything including discourse, right? If not, who makes the decision so I can talk to?

Reply:
discourse.flox.dev is now live